recipes/redradist-icc/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.0.0":
+    sha256: d4beade372e909b06d17450646d2d47cf43316d5903191338dcadffbdcf9f16d
+    url: https://github.com/redradist/Inter-Component-Communication/archive/refs/tags/v1.0.0.zip
```

Comment:
Theres a preference for tar.gz since its smaller

Reply:
Done, see in upcoming changes